### DISCUSSION TITLE
RIR "IPv4/6 Stats" button without function?

### DISCUSSION THREAD

**Discussion by Serazio** [netbox-community/netbox#6630] [General]
Hi,

could anyone explain the function of the "IPv4/6 Stats" Button in IPAM -> RIRs?

I can't find anything related to this button. The template just changes the button text and includes the text "Numbers shown indicate /64 prefixes." 

Regards
Alex

**Comment by jeremystretch** [comment]
This was an old function of the RIRs table that was removed in #5139. Looks like we neglected to clean up the template. I'll clean it up for the next release.
